playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -130,7 +131,7 @@ private void startPlaybackFrom(long startPosition) {
         }
     }
 
-    public View getView() {
+    public PKView getView() {
```

Comment:
The view returned here should be a ViewGroup created by PlayerController. It will hold the ExoPlayerView OR the VideoView.